### PR TITLE
armory change the musical but it actually works this time

### PR DESCRIPTION
**Pull request by bearrrrrrrr**
i literally want to edie

jsut merge it, please. we all know what it does. you have the last PR. just dfo it. please. holy fuck. im so mad and depressed and angry. 24h of my life down the drainl. why do i live

### CODE REVIEW

File: code/modules/research/designs/designs_weapon.dm [hunk 1]
```diff
@@ -174,29 +174,45 @@
...
+	req_tech = list(TECH_MATERIAL = 1, TECH_COMBAT = 1)
+	materials = list(MATERIAL_STEEL = 15000, MATERIAL_PLASTIC = 5000, MATERIAL_ALUMINUM = 10000, MATERIAL_OSMIUM = 10000)
+	build_path = /obj/item/weapon/gun/projectile/shotgun/pump/combat
+	sort_string = "TAPAD"
+
+/datum/design/item/weapon/lasercarbinerifle
+	name = "laser carbine"
+	id = "lasercarbinerifle"
+	materials = list(MATERIAL_STEEL = 5000, MATERIAL_PLASTIC = 500, MATERIAL_ALUMINUM = 5000, MATERIAL_GLASS = 1500)
+	req_tech = req_tech = list(TECH_MATERIAL = 1, TECH_COMBAT = 1)
```
Comment (MarinaGryphon) [inline code/modules/research/designs/designs_weapon.dm:212]:
```suggestion
	req_tech = list(TECH_MATERIAL = 1, TECH_COMBAT = 1)
```

**Comment by bearrrrrrrr** [comment]
it didn't work this time

FIXIIIIIIIIIIIIIIIING-

File: code/modules/research/designs/designs_weapon.dm [hunk 2]
```diff
@@ -174,29 +174,45 @@
 	sort_string = "TAJAB"
 
 /datum/design/item/weapon/wt550
-	name = "9mm submachine gun"
+	name = "WT550 submachine gun"
 	id = "wt550"
-	req_tech = list(TECH_COMBAT = 5, TECH_MATERIAL = 2)
-	materials = list(MATERIAL_STEEL = 8000, MATERIAL_SILVER = 3000, MATERIAL_DIAMOND = 1500)
```
Comment (AdamElTablawy) [inline code/modules/research/designs/designs_weapon.dm:180]: why did you make it easier to print

**Comment by AdamElTablawy** [comment]
again, reminder that the materials limits aren't a limit here

8000 plastic is 4 sheets

sec can and will just order a 50 sheet crate from cargo, which will be approved, because cargo will literally never deny an order from security because they have nothing to do or they'd just ask a borg to do it or they'd just ask command to do it

make use of rarer materials that can be ordered in 10 sheet increments instead and you'd at least force cargo to spend more points on it, since your intent is to create scarcity

otherwise this genuinely does buff security since the only restriction for guns vanishes in literal 50 sheet crates

i mentioned the same thing to jade in general and they agreed, osmium or something is nice and limited while making some level of logical sense. otherwise this means sec can shit out as many guns as they want

**Comment by BurgerLUA** [comment]
no clue what's wrong with travis, please submit a dummy commit to see if this shit would work

**Comment by AdamElTablawy** [comment]
There is still no fix for the gun printing problem and it requires admins to stop sec from printing 50 guns at roundstart.

The very same thing you said wouldn't happen is, and requires administrative enforcement to stop.

Limit gun printing on code green, at the absolute least.

**Comment by BurgerLUA** [comment]
Closing this PR as there are no recent changes and it's on the second page.

**Pull request closed without merging**